### ISSUE TITLE
Notebook theme colors review

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#125549]
After we introduced more customizations of the notebook layout, the theme colors for notebook don't cover all the scenarios, we should review and consolidate them to ensure they work for all combinations of settings.

List of existing theme colors:

* `"notebook.cellEditorBackground"`
* `"notebook.cellToolbarSeparator"`
* `"notebook.focusedEditorBorder"`
* `"notebook.focusedRowBorder"`
* `"notebook.outputContainerBackgroundColor"`
* `"notebook.cellBorderColor"`
* `"notebook.cellInsertionIndicator"`
* `"notebook.cellStatusBarItemHoverBackground"`
* `"notebook.focusedCellBorder"`
* `"notebook.inactiveFocusedCellBorder"`
* `"notebook.rowHoverBackground"`
* `"notebook.selectedCellBackground"`
* `"notebook.selectedCellBorder"`
* `"notebook.symbolHighlightBackground"`
* `"notebookScrollbarSlider.activeBackground"`
* `"notebookScrollbarSlider.background"`
* `"notebookScrollbarSlider.hoverBackground"`
* `"notebookStatusErrorIcon.foreground"`
* `"notebookStatusRunningIcon.foreground"`
* `"notebookStatusSuccessIcon.foreground"`

**Comment by tanhakabir** [comment]
I'd vote for having the cell background color to be gray and the output to be white by default because currently with some outputs that are white, there's a weird gray thick border around it due to padding.

**Comment by misolori** [comment]
For output background color, I've actually made it transparent (`"notebook.outputContainerBackgroundColor": "#0000"`) in my user settings which tends to look a little better with charts + tables:

## Before
<img width="1035" alt="CleanShot 2021-06-09 at 09 58 12@2x" src="https://user-images.githubusercontent.com/35271042/121397484-3968f300-c909-11eb-93e7-fa61ca3e0dc2.png">

<img width="788" alt="CleanShot 2021-06-09 at 10 02 14@2x" src="https://user-images.githubusercontent.com/35271042/121398036-c8760b00-c909-11eb-8a00-72b567bf62c7.png">


## After
<img width="1026" alt="CleanShot 2021-06-09 at 09 58 40@2x" src="https://user-images.githubusercontent.com/35271042/121397555-4c7bc300-c909-11eb-8d4d-44d98dea72fe.png">

<img width="779" alt="CleanShot 2021-06-09 at 10 03 17@2x" src="https://user-images.githubusercontent.com/35271042/121398194-f2c7c880-c909-11eb-957e-867d2d1bf865.png">

**Comment by tanhakabir** [comment]
Maybe transparent by default is a good idea?

**Comment by rebornix** [comment]
👍 for having transparent output background with a gray cell editor background.

**Comment by misolori** [comment]
For cell editor bg, I think at one point we did use a bg color and then went away from that because of the accessibility issues it introduced with color contrast?

**Comment by tanhakabir** [comment]
The color contrast between gray background and the code?

**Comment by misolori** [comment]
Yes, because the all of the color syntax colors were create to work against the editor background color. Dark theme doesn't has as many issues as the light theme. On light (if I'm remembering correctly) it just passes the color contrast ratio. Here's a sample of the CC ratios when using a faint gray bg:

![image](https://user-images.githubusercontent.com/35271042/121556077-6a5c2d00-c9c8-11eb-8b8f-6236f858813e.png)
![image](https://user-images.githubusercontent.com/35271042/121556090-6d571d80-c9c8-11eb-8a7e-353e07b0eb5d.png)
![image](https://user-images.githubusercontent.com/35271042/121556097-6e884a80-c9c8-11eb-8d30-52cc207a99e6.png)

**Comment by tanhakabir** [comment]
I saw in the survey responses that several had complained that it's difficult to see where code cells are and where markdown cells are. I think they want/are used to code cell backgrounds being slightly gray and not the same color

**Comment by claudiaregio** [comment]
This feedback just came in, wanted to add onto here. 

![image](https://user-images.githubusercontent.com/19276747/122836242-ba7b9f00-d2a6-11eb-8876-268f2d2bc8c1.png)

**Comment by claudiaregio** [comment]
![image](https://user-images.githubusercontent.com/19276747/122842423-34fdec00-d2b2-11eb-9535-5aa98e5bbb03.png)

**Comment by claudiaregio** [comment]
![image](https://user-images.githubusercontent.com/19276747/123490952-53255e00-d5ca-11eb-8260-4820d7847a73.png)

**Comment by Kevin-Delnoije** [comment]
Hey i am having the issue @claudiaregio  mentioned above.
With the gruvbox theme (dark variant) it is hard to see currently active cell see image:
![Screenshot from 2021-06-28 17-07-23](https://user-images.githubusercontent.com/45107767/123661621-fb3e5f80-d834-11eb-96fe-8c0616f537c4.png)

I cannot find a setting that changes the highlight color, any suggestions?

**Comment by roblourens** [comment]
@Kevin-Delnoije You can set

```
"workbench.colorCustomizations": {
        "notebook.focusedCellBorder": "#ff0000"
}
```

for example

**Comment by roblourens** [comment]
Should the 3 of us get together sometime or is this a good October issue?
